packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -285,6 +291,13 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 		},
 	},
 	actions: {
+		clearNodeOutgoingConnections(nodeName: string) {
```

Comment:
We already have a similar action for deleting node connections: `removeAllNodeConnection` perhaps we could just  add a optional config object to configure it only to delete specific connections:
```ts
removeAllNodeConnection(
			node: INodeUi,
			{ preserveInputConnections = false, preserveOutputConnections = false } = {},
		): void {
			const uiStore = useUIStore();
			uiStore.stateIsDirty = true;
			// Remove all source connections
			if (!preserveOutputConnections && this.workflow.connections.hasOwnProperty(node.name)) {
				delete this.workflow.connections[node.name];
			}

			// Remove all destination connections
			if (preserveInputConnections) return;
			const indexesToRemove = [];
			let sourceNode: string,
				type: string,
				sourceIndex: string,
				connectionIndex: string,
				connectionData: IConnection;
			for (sourceNode of Object.keys(this.workflow.connections)) {
				for (type of Object.keys(this.workflow.connections[sourceNode])) {
					for (sourceIndex of Object.keys(this.workflow.connections[sourceNode][type])) {
						indexesToRemove.length = 0;
						for (connectionIndex of Object.keys(
							this.workflow.connections[sourceNode][type][parseInt(sourceIndex, 10)],
						)) {
							connectionData =
								this.workflow.connections[sourceNode][type][parseInt(sourceIndex, 10)][
									parseInt(connectionIndex, 10)
								];
							if (connectionData.node === node.name) {
								indexesToRemove.push(connectionIndex);
							}
						}

						indexesToRemove.forEach((index) => {
							this.workflow.connections[sourceNode][type][parseInt(sourceIndex, 10)].splice(
								parseInt(index, 10),
								1,
							);
						});
					}
				}
			}
		}
```
And then it can be called directly like this: `this.workflowsStore.removeAllNodeConnection(node);` from the component, so wouldn't need to add another action. WDYT?